Marlin/Configuration.h

```diff
@@ -351,7 +351,8 @@
  * --NORMAL IS 4.7kohm PULLUP!-- 1kohm pullup can be used on hotend sensor, using correct resistor and table
  *
  * Temperature sensors available:
- *
+ *    
+ *    -5 : thermocouple with MAX31865 (only for sensor 0)
  *    -4 : thermocouple with AD8495
  *    -3 : thermocouple with MAX31855 (only for sensor 0)
  *    -2 : thermocouple with MAX6675 (only for sensor 0)
```

Comment:
Aren't we now able to use the SPI sensors on all channels, not only "sensor 0"?

Reply:
Haven't tried it, I don't know why it was limited to only sensor 0. I'll dig some more.

Reply:
For this type, just 0 and 1 currently.